test/helpers/utils.go

```diff
@@ -699,3 +699,14 @@ func DualStackSupportBeta() bool {
 
 	return GetCurrentIntegration() == "" && supportedVersions(k8sVersion)
 }
+
+// CiliumEndpointSliceFeatureEnabled returns true only if the environment has a kubernetes version
+// greater than or equal to 1.21.
+func CiliumEndpointSliceFeatureEnabled() bool {
+	k8sVersionGreaterThan121 := versioncheck.MustCompile(">=1.21.0")
```

Comment:
Please make this consistent, previously it was correct but misleading. Now the names and the numbers don't match.

Either:
```suggestion
	k8sVersionGreaterThan121 := versioncheck.MustCompile(">1.21.0")
```

or:
```suggestion
	k8sVersionGreaterEqual121 := versioncheck.MustCompile(">=1.21.0")
```

Reply:
my bad, updated now.